docs/guides/chip_tool_guide.md

```diff
@@ -410,10 +414,14 @@ The CHIP Tool can run in one of the following modes:
     [1650992689511] [32397:1415601] CHIP: [TOO] Run command failure: ../../../examples/chip-tool/commands/common/CHIPCommand.cpp:392: CHIP Error 0x00000032: Timeout
     ```
 
+    Moreover, when using the single-command mode, the CHIP Tool will establish a
+    new CASE session with every command sent (and close the existing one).
```

Comment:
It will not close or expire anything.  It will just create a new CASE session.

Reply:
Removed parenthesis.